source/guide_focalboard.rst

```diff
@@ -0,0 +1,119 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: web
+.. tag:: project-management
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/focalboard.svg
+      :align: center
+
+##########
+Focalboard
+##########
+
+.. tag_list::
+
+Focalboard_ is an open source, self-hosted alternative to Trello, Notion, and Asana. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+It's recommended to use a PostgreSQL database. Install and configure PostgreSQL server as described in the :lab:`PostgreSQL guide <guide_postgresql>`.
+
+Installation
+============
+
+Download the latest release_ to your home directory and extract the application.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ wget https://github.com/mattermost/focalboard/releases/download/v0.6.1/focalboard-linux.tar.gz
+ [isabell@stardust ~]$ tar --extract --gzip --file focalboard-linux.tar.gz
+ [isabell@stardust ~]$ rm focalboard-linux.tar.gz
+ [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+You need to modify ``~/focalboard-app/config.json``:
+ * Change ``"localOnly": true,`` to ``"localOnly": false,``
+ * Add ``"session_expire_time": 2592000,``
+ * Add ``"session_refresh_time": 18000,``
+ 
+If PostgreSQL is used, the database configuration also needs to be adjusted. Edit the database config in the file ``~/focalboard-app/config.json``. These two values must be changed:
+.. warning:: Replace ``<db_user>``, ``<db_password>`` and ``<db_name>`` with your values.
+
+.. code-block:: json
```

Comment:
The formatting for this and the following code block [is broken](https://deploy-preview-966--uberlab.netlify.app/guide_focalboard.html#configuration) because of a missing blank line after the `.. code-block` one.